pages/docs/tutorials/validate-documents.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Validate AsyncAPI document with Studio"
+description: This tutorial will teach you how to validate AsyncAPI documents using the AsyncAPI Studio tool.
+weight: 120
+---
+
+## Introduction
+This tutorial will teach you how to validate AsyncAPI documents using the [AsyncAPI Studio tool](https://studio.asyncapi.com/).
+
+You will start with a broken AsyncAPI document and troubleshoot via console errors step-by-step until we end up with a valid AsyncAPI document. This process will illustrate how to identify [`REQUIRED` properties in AsyncAPI documents](https://www.asyncapi.com/docs/reference/specification/v2.5.0#A2SObject).
```

Comment:
```suggestion
You will start with a broken AsyncAPI document and troubleshoot via console errors step-by-step until we end up with a valid AsyncAPI document. This process will illustrate how to identify [`REQUIRED` properties in AsyncAPI documents](https://www.asyncapi.com/docs/reference/specification/latest#A2SObject).
```

@alequetzalli @derberg shouldn't the link for the specification be directed to the latest specification, instead of just pointing to v2.5.0